src/headers/constants.rs

```diff
@@ -147,6 +147,9 @@ pub const SERVER: HeaderName = HeaderName::from_lowercase_str("server");
 ///  The `Server` Header
 pub const SERVER_TIMING: HeaderName = HeaderName::from_lowercase_str("server-timing");
 
+///  The `SourceMap` Header
+pub const SOURCE_MAP: HeaderName = HeaderName::from_lowercase_str("sourcemap");
```

Comment:
Should this be `SOURCEMAP`?

Reply:
The canonical spelling of the header is `SourceMap` -- which is not quite `Source-Map` but I think `SOURCE_MAP` is the right capitalization.